File: Marvellous_LB/8-JAVA/program7.java
import java.util.*;
class Numbers
{
    public void EvenFactors(int iVal)
    {
        System.out.println("Even Factors are : ");
        int iCnt=0;
        for(iCnt=2;iCnt<=(iVal/2);iCnt+=2)
        {
            if(iVal%2==0)
            {
                System.out.println(iCnt);
            }
        }
    }
}
public class program7
{
    public static void main (String arg[])
    {
        int iNo=0;
        Scanner sobj=new Scanner(System.in);

        System.out.println("Enter number to check even Factorials : ");
        iNo=sobj.nextInt();

        Numbers nobj=new Numbers();
        nobj.EvenFactors(iNo);
    }
}

File: Marvellous_LB/4-Pattern printing/program7.c
#include<stdio.h>

void Display(int iVal)
{
    int iCnt=0;
    char ch ='a';

    if(iVal<0)
    {
        iVal=-iVal;
    }

    for(iCnt=1;iCnt<=iVal;iCnt++,ch++)
    {
        printf("%c\t",ch);
    }
}

int main()
{
    int iNo=0;

    printf("Enter Number you want to print pattern:\t");
    scanf("%d",&iNo);

    Display(iNo);

    return 0;
}